src/server/rest/v1/validator/ChargingStationValidator.ts

```diff
@@ -139,6 +141,12 @@ export default class ChargingStationValidator extends SchemaValidator {
     return data;
   }
 
+  public validateChargingStationGetDiagnostics(data: HttpChargingStationGetDiagnostics): HttpChargingStationGetDiagnostics {
```

Comment:
Missing Request at the end of the HttpChargingStationGetDiagnostics like the others ;-)